README.md

```diff
@@ -57,10 +57,6 @@ release](https://github.com/lifeomic/cli/releases).
 Run `lo setup` to configure the default environment and account you wish to use.
 You can later override the default account using the `-a` option for commands.
 
-#### Dev account access
```

Comment:
Our dev setup should not be in this public repo, because we don't want to advertise it's existence and possibly confuse customers.